### DISCUSSION TITLE
V400 VERS V600

### DISCUSSION THREAD

**Discussion by Mike64p** [Guilouz/Klipper-Flsun-Speeder-Pad#143] [General]
Bonjour,
Je viens de flasher ma V400 avec la méthode dite "GUILOUZ", et je voulais savoir si je pouvais faire les mise à jour vers V600 ou bien ce n'est plus possible?
Merci d'avance.

**Comment by Guilouz** [comment]
Une V600 n'existe pas... Il faut arrêter de croire ce que une certaine personne annonce sur les réseaux sociaux...
La mise à jour 1.4 n'apporte rien de plus à part des configurations modifiées pour atteindre les 600 mm/s et le fait que l'accès SSH ne soit plus possible, mes configurations sont modifier en conséquence.

**Comment by Mike64p** [comment]
Ok très bien, pas de souci, je ne voulais offenser personne, et je ne vois
absolument pas de qui tu parles désolé.
Ce sont des infos prises sur le net et certains sites spécialisés.

Bref, donc avec ta configuration est il possible d'utiliser un profil de
slicer pour ces vitesses alors ?

Merci.


Le mer. 29 nov. 2023, 12:21, Cyril Guislain ***@***.***> a
écrit :

**Comment by Guilouz** [comment]
Oui notamment la société Whanao qui rebrand les V400 en V600 c’est n’importe quoi…

Oui Klipper est très malléable, ce sont juste des configurations qui change le fonctionnement de la machine.
Mes fichiers de configuration permettent d’attendre les 600mm/s sans problème sans avoir besoin de passer sur le firmware 1.4 qui bloque l’accès SSH.

**Reply by Mike64p** [reply]
Merci et bonne soirée.
